src/index.ts

```diff
@@ -97,11 +97,14 @@ class SandpackInstance {
       this.resizePollingTimer = setInterval(this.sendResizeEvent, 300);
     };
 
+    resizePolling();
+
     /**
-     * Ideally we should use a `MutationObserver` to trigger a resize event,
-     * however, we noted that it's not reliable, so we went for polling strategy
+     * Ideally we should only use a `MutationObserver` to trigger a resize event,
+     * however, we noted that it's not 100% reliable, so we went for polling strategy as well
      */
-    resizePolling();
+    const observer = new MutationObserver(this.sendResizeEvent);
```

Comment:
Should this have a debounce?

Reply:
I would throttle, actually, in order to immediately call the callback